src/MongoDB.Driver.Core/Core/Operations/ReadCommandOperation.cs

```diff
@@ -32,8 +29,11 @@ namespace MongoDB.Driver.Core.Operations
     /// Represents a read command operation.
     /// </summary>
     /// <typeparam name="TCommandResult">The type of the command result.</typeparam>
-    public class ReadCommandOperation<TCommandResult> : CommandOperationBase<TCommandResult>, IReadOperation<TCommandResult>
+    public class ReadCommandOperation<TCommandResult> : CommandOperationBase<TCommandResult>, IReadOperation<TCommandResult>, IRetryableReadOperation<TCommandResult>
```

Comment:
As it turned out this is the only operation that actually implements IRetryableReadOperation.

All other retryable operations end up delegating to this one.